Makefile

```diff
@@ -5,7 +5,7 @@ NAMESPACE?=prometheus-operator-e2e-tests-$(shell LC_CTYPE=C tr -dc a-z0-9 < /dev
 PROMU := $(GOPATH)/bin/promu
 PREFIX ?= $(shell pwd)
 
-CLUSTER_IP?=$(shell minikube ip)
+CLUSTER_IP?=$(shell kubectl config view --minify | grep server: | cut -f 3 -d ":" | tr -d "//")
```

Comment:
For the reference: may want to run locally against other clusters than minikube therefore a simple "if CI" won't work, so skipping this for now.